### DISCUSSION TITLE
Actions are taking several minutes before kicking off

### DISCUSSION THREAD

**Discussion by cnelms** [community/community#57453] [Actions] [answered]
### Select Topic Area

Bug

### Body

Hello,
My actions are at https://github.com/cai-sf/deluxe/actions and they are taking forever to start, and actually just noticed that my last 3 pushes haven't even started their workflows and its been around 30 minutes. Usually they start within seconds after a push so something is wrong.
Thanks,
Carey
[redacted due to PII]

**Comment by Velx098765** [comment]
Thanks, I thought something like that was happening before, and I finally have closure instead of thinking I was crazy.
